pom.xml

```diff
@@ -100,6 +100,7 @@
         <!-- Potentially used by downstream projects -->
         <slf4j-api.version>${slf4j.version}</slf4j-api.version>
         <slf4j.version>1.7.36</slf4j.version>
+        <log4j-slf4j-impl.version>2.24.3</log4j-slf4j-impl.version>
```

Comment:
Shouldn't we just have a `log4j2.version`?

Reply:
Yes, that'd be better